flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchSinkBase.java

```diff
@@ -402,6 +405,18 @@ protected BulkProcessor buildBulkProcessor(BulkProcessor.Listener listener) {
         return bulkProcessorBuilder.build();
     }
 
+    private void configureBulkSize(BulkProcessor.Builder bulkProcessorBuilder) {
```

Comment:
nit: I would vote for code consistency with `bulkProcessorFlushIntervalMillis` here by removing this method?

Reply:
Honestly, I'd rather extract a method for `bulkProcessorFlushIntervalMillis`. I feel strong about long methods. Personally I like when methods have a limited responsibilities. When reading the `buildBulkProcessor` we do not need to think how exactly is the bulk size configured. WDYT?

Reply:
sure, extracting another method is also fine